### PR TITLE
Drop DRF from dependencies

### PR DESCRIPTION
**Pull request by menecio**
DRF was used for deserializing to JSON, this can be achieved by using standard python.

### CODE REVIEW

**Comment by menecio** [comment]
Depends on #2

File: rele/client.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def __init__(self):
             credentials=settings.RELE_GC_CREDENTIALS)
 
     def publish(self, topic, data, **attrs):
-        data = JSONRenderer().render(data)
+        data = json.dumps(data).encode()
```
Comment (andrewgy8) [inline rele/client.py]: Im afraid this wont work in the case that you pass in `serializers.data`. We tried this, and thats why we had to use `data = JSONRenderer().render(data)`. If anything, you will would have to pass in your data already encoded to the publish function, or we make the encoding method configurable.

**Comment by tobami** [comment]
To be implemented here: https://github.com/mercadona/rele/issues/8

**Comment by menecio** [comment]
@tobami why did you close this one without having an agreement on how to proceed? Because DRF compat can be achieved right now without doing anything fancy, just by know that you must use `.to_representation()` method when publishing data

File: rele/client.py (hunk 1 above)
Comment (menecio) [reply rele/client.py]: Did you try passing `serializer.to_representation()` ?

**Pull request closed without merging**